backend/src/test/java/de/laura/project/controller/TriviaControllerTest.java

```diff
@@ -39,45 +45,104 @@ private String getUrl() {
+
+        //THEN
+        assertThat(response.getStatusCode(), is(HttpStatus.OK));
+        assertTrue(response.getBody());
+    }
+
+    @Test
+    @DisplayName("Checks whether user's incorrect answer returns false")
+
+    public void checkAnswerIsFalse() {
+
+        //GIVEN
+        List<String> answerForTriviaQuestionSet = List.of("correct answer", "answerOne", "answerTwo", "answerThree");
+        List<String> anotherAnswerForTriviaQuestionSet = List.of("another correct answer", "answerOne", "answerTwo", "answerThree");
+
+        tempTriviaQuestionDB.getTriviaQuestionSetList().add(new TriviaQuestionSet(1, "This is an awesome question", "correct answer", answerForTriviaQuestionSet));
+        tempTriviaQuestionDB.getTriviaQuestionSetList().add(new TriviaQuestionSet(2, "This is another awesome question", "another correct answer", anotherAnswerForTriviaQuestionSet));
+
+        TriviaAnswerDTO triviaAnswerDTO = new TriviaAnswerDTO(1, "answerOne");
+
+        //WHEN
+        ResponseEntity<Boolean> response = testRestTemplate.postForEntity(getUrl() + "/answer", triviaAnswerDTO, Boolean.class);
+
+        //THEN
+        assertThat(response.getStatusCode(), is(HttpStatus.OK));
+        assertFalse(response.getBody());
+    }
+
+
+    @Test
+    @DisplayName("Calls question based on questionsID")
+
+    public void getSingleQuestion() {
+
+        //GIVEN
+        List<String> answerForTriviaQuestionSet = List.of("correct answer", "answerOne", "answerTwo", "answerThree");
+        List<String> anotherAnswerForTriviaQuestionSet = List.of("another correct answer", "answerOne", "answerTwo", "answerThree");
+
+        tempTriviaQuestionDB.getTriviaQuestionSetList().add(new TriviaQuestionSet(1, "This is an awesome question", "correct answer", answerForTriviaQuestionSet));
+        tempTriviaQuestionDB.getTriviaQuestionSetList().add(new TriviaQuestionSet(2, "This is another awesome question", "another correct answer", anotherAnswerForTriviaQuestionSet));
+
         //WHEN
-        ResponseEntity<TriviaQuestionSet[]> response = testRestTemplate.getForEntity(getUrl(), TriviaQuestionSet[].class);
+        ResponseEntity<TriviaQuestionSetWithoutCorrectAnswer> response = testRestTemplate.getForEntity(getUrl() + "/2", TriviaQuestionSetWithoutCorrectAnswer.class);
 
         //THEN
-        assertThat(response.getBody(), arrayContainingInAnyOrder(
-                new TriviaQuestionSet(1, "This is an awesome question", "correct answer", answerForTriviaQuestionSet),
-                new TriviaQuestionSet(2, "This is another awesome question", "another correct answer", anotherAnswerForTriviaQuestionSet)));
+        assertThat(response.getStatusCode(), is(HttpStatus.OK));
+        assertThat(response.getBody(), is(new TriviaQuestionSetWithoutCorrectAnswer(2,"This is another awesome question", anotherAnswerForTriviaQuestionSet)));
     }
 }
```

Comment:
you may want to consider including a test that requests a question set that is not available in your data base